src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -343,6 +346,12 @@ void FailSafeCleanup(const chip::DeviceLayer::ChipDeviceEvent * event)
             ChipLogError(Zcl, "OpCreds: failed to delete fabric at index %u: %" CHIP_ERROR_FORMAT, fabricIndex, err.Format());
         }
     }
+
+    if (nocAddedOrUpdatedDuringFailsafe)
```

Comment:
For the "added" case, did this not get handled by OnFabricRemoved when we did the DeleteFabricFromTable above?

So here, I think we only need to do it for the "updated" case, not for "added or updated"...

Reply:
Good catch. It was indeed superfluous in the add case.